### ISSUE TITLE
Save profile triggers reslice

### ISSUE THREAD

**Issue by BagelOrb** [Ultimaker/Cura#5014]
**Application Version**
Current master (fresh redesign) 13 dec 1028

**Platform**
Linux

**Printer**
UM3

**Steps to Reproduce**
Make setting changes
save as profile
make setting changes
slice
[Update profile with current settings/overrides]

**Actual Results**
slice data is thrown away

**Expected results**
Slice data remains

**Additional Information**
I like to save all changes as often as possible so that I don't loose any data, but if that means I have to reslice then I wouldn't save as often and I could forget saving altogether, which might lead to problems later when I come back to my current project.

**Comment by nallath** [comment]
Yeah, this is unfortunately quite hard to fix because of the way profiles work.

**Comment by GregValiant** [comment]
I'm cleaning up these old reports.  I'll go ahead and close this.  If the problem has persisted into 5.13.0 please make out a new report.
